DotNet/RoadmapLogic/RoadmapImage.cs

```diff
@@ -9,6 +10,11 @@ namespace RoadmapLogic
 {
     public static class RoadmapImage
     {
+        private const string  s_MissingProjectMessageText = "Alert! Project(s) have been omitted from the roadmap. Please Review.";
+        private const string  s_BeforeStartQuarterMessage = "Project(s) before start quarter was(re) omitted.";
+        private const string  s_AfterLastQuarterMessage = "Project(s) after last quarter was(re) omitted.";
+        private const string s_OutsideQuarterRangeMessage = "Projects before start and after last quarters were omitted.";
```

Comment:
What's this "was(re)" thing? Just display things in the plural to avoid parentheses as well. I like Peter's suggestions to use "window" of time instead of the more technical "start quarter".

Reply:
Was(re) represents that it could be one or more projects which I followed based on the original message where we used "Project(s). If we show plural, it would be misleading as there could be only one project that was outside the start and last quarters.

Reply:
But it doesn't read well. I don't think "was(re)" is correct English. It would be better to just always show the message in the plural.